tools/install_openblas_i686.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+wget https://github.com/xianyi/OpenBLAS/releases/download/v0.3.21/OpenBLAS-0.3.21.zip
+unzip OpenBLAS-0.3.21.zip
+cd OpenBLAS-0.3.21
+make TARGET=NORTHWOOD
```

Comment:
Why northwood? What if I want to run on the AMD K6-2 which I had before I got my Pentium 4 back in 2003?

Reply:
At least, I needed to set `TARGET` to build to work around this error in OpenBLAS build on i686:
```
strmm_kernel_LN.o: No such file or directory
```
Maybe the `TARGET` should be old enough, like `P2`.

Reply:
I found at least KATMAI is needed to build openblas https://github.com/Qiskit/qiskit-aer/pull/1653/commits/f410260547086d7c1eea5fb1ad11c75c9d8518d8.

Reply:
I guess we need SSE or MMX to run openblas. Hopefully nobody tries to run aer with a CPU >23 yrs old :)